modules/tinymce/src/plugins/imagetools/main/ts/ui/Buttons.ts

```diff
@@ -12,48 +12,54 @@ import * as Actions from '../core/Actions';
 const register = (editor: Editor) => {
   const cmd = (command: string) => () => editor.execCommand(command);
 
+  const onSetup = (buttonApi) => {
```

Comment:
Make sure you put a type in for `buttonApi`

Reply:
FIxed.